projects/1.20/assets/slimy-stuff/slimy_stuff/lang/zh_cn.json

```
@@ -0,0 +1,130 @@
+{
+  "biome.slimy_lands.slimy_lands": "史莱姆之地",
+  "block.slimy_lands.liquid_slime": "液体史莱姆",
+  "block.slimy_lands.slime_bump": "不可触碰的史莱姆",
+  "block.slimy_lands.slime_fence": "史莱姆栅栏",
+  "block.slimy_lands.slime_fence_gate": "史莱姆栅栏门",
+  "block.slimy_lands.slime_log": " 史莱原木",
+  "block.slimy_lands.slime_planks": "史莱木板",
+  "block.slimy_lands.slime_pressure_plate": "史莱姆压力板",
+  "block.slimy_lands.slime_puddle": "史莱姆坑",
+  "block.slimy_lands.slime_sapling": "史莱姆树苗",
+  "block.slimy_lands.slime_slab": "史莱姆板",
+  "block.slimy_lands.slime_stairs": "史莱姆楼梯",
+  "block.slimy_lands.slime_wood": "史莱木",
+  "block.slimy_lands.slimy_grass": "史莱姆草方块",
+  "block.slimy_stuff.caramelized_slime_block": "焦糖史莱姆方块",
+  "block.slimy_stuff.caramelized_slime_pressure_plate": "焦糖史莱姆压力板",
+  "block.slimy_stuff.caramelized_slime_slab": " 焦糖史莱姆台阶",
+  "block.slimy_stuff.caramelized_slime_stairs": "焦糖史莱姆楼梯",
+  "block.slimy_stuff.jug": "罐子",
+  "block.slimy_stuff.jug_brain_coral": "脑珊瑚罐子",
+  "block.slimy_stuff.jug_bubble_coral": "泡泡珊瑚罐子",
+  "block.slimy_stuff.jug_fire_coral": "火纹珊瑚罐子",
+  "block.slimy_stuff.jug_horn_coral": "角珊瑚罐子",
+  "block.slimy_stuff.jug_lilac": "丁香罐子",
+  "block.slimy_stuff.jug_peony": "牡丹罐子",
+  "block.slimy_stuff.jug_rose_bush": "玫瑰丛罐子",
+  "block.slimy_stuff.jug_sunflower": "向日葵罐子",
+  "block.slimy_stuff.jug_tube_coral": "管珊瑚罐子",
+  "block.slimy_stuff.pressure_plate_wall": "焦糖史莱姆墙",
+  "block.slimy_stuff.slime_bump": "不可触碰的史莱姆",
+  "block.slimy_stuff.slime_puddle": "史莱姆坑",
+  "effect.bounce": "弹性",
+  "effect.fire_aura": "火焰光环",
+  "effect.gummy": "粘性的",
+  "effect.slimy_stuff.bounce": "弹性",
+  "effect.slimy_stuff.fire_aura": "火焰光环",
+  "effect.slimy_stuff.gummy": "粘性",
+  "effect.slimy_stuff.sticky": "粘性",
+  "effect.sticky": "粘性",
+  "effect.sticky_resistance": "粘性阻力",
+  "fluid.slimy_lands.liquid_slime": "液态史莱姆",
+  "item_group.slimy_stuff.slimy_lands_tab": "史莱姆小玩意",
+  "item.minecraft.lingering_potion.effect.slime_potion": "滞留型史莱姆药水",
+  "item.minecraft.potion.effect.slime_potion": "史莱姆药水",
+  "item.minecraft.splash_potion.effect.slime_potion": "喷溅型史莱姆药水",
+  "item.minecraft.tipped_arrow.effect.slime_potion": "史莱姆箭",
+  "item.slimy_lands.bottled_slime": "瓶装史莱姆",
+  "item.slimy_lands.bottled_slime_2": "瓶装史莱姆",
+  "item.slimy_lands.caramelized_slime": "焦糖史莱姆",
+  "item.slimy_lands.glass_jar": "玻璃罐子",
+  "item.slimy_lands.liquid_slime_bucket": "液体史莱姆桶",
+  "item.slimy_lands.slimy_boots": "史莱姆靴子",
+  "item.slimy_lands.slimy_chestplate": "史莱姆头盔",
+  "item.slimy_lands.slimy_dagger": "史莱姆匕首",
+  "item.slimy_lands.slimy_helmet": "史莱姆头盔",
+  "item.slimy_lands.slimy_lands_icon": "史莱姆之地图标",
+  "item.slimy_lands.slimy_leggings": "史莱姆绑腿",
+  "item.slimy_lands.suspicious_jam_blindness_1": "可疑果酱",
```

```suggestion
  "item.slimy_lands.suspicious_jam_blindness_1": "谜之果酱",
```
参见原版`谜之炖菜`/`Suspicious Stew`，请全文修改